2022-11-15 16:51:17,581 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 16:51:17,582 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 16:51:17,584 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 16:51:17,584 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 16:51:17,586 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 16:51:17,586 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 16:51:17,587 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165117-2zw4zfjp/logs/debug.log
2022-11-15 16:51:17,589 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165117-2zw4zfjp/logs/debug-internal.log
2022-11-15 16:51:17,591 INFO    MainThread:1637445 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f171613fb00>
2022-11-15 16:51:17,593 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 16:51:17,595 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 16:51:17,597 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 360bszfx
2022-11-15 16:51:17,600 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/360bszfx
2022-11-15 16:51:17,600 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 16:51:17,602 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 16:51:17,604 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 16:51:21,720 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 16:51:21,721 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 16:51:21,729 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-11-15 16:51:21,735 INFO    MainThread:1637445 [wandb_init.py:init():569] starting backend
2022-11-15 16:51:21,736 INFO    MainThread:1637445 [wandb_init.py:init():573] setting up manager
2022-11-15 16:51:21,744 INFO    MainThread:1637445 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-15 16:51:21,750 INFO    MainThread:1637445 [wandb_init.py:init():580] backend started and connected
2022-11-15 16:51:21,774 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-15 16:51:21,775 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-15 16:51:21,776 INFO    MainThread:1637445 [wandb_init.py:init():658] updated telemetry
2022-11-15 16:51:21,844 INFO    MainThread:1637445 [wandb_init.py:init():694] communicating run to backend with 60 second timeout
2022-11-15 16:51:23,182 INFO    MainThread:1637445 [wandb_run.py:_on_init():2000] communicating current version
2022-11-15 16:51:23,304 INFO    MainThread:1637445 [wandb_run.py:_on_init():2004] got version response 
2022-11-15 16:51:23,304 INFO    MainThread:1637445 [wandb_init.py:init():728] starting run threads in backend
2022-11-15 16:51:28,063 INFO    MainThread:1637445 [wandb_run.py:_console_start():1980] atexit reg
2022-11-15 16:51:28,063 INFO    MainThread:1637445 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-15 16:51:28,064 INFO    MainThread:1637445 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-15 16:51:28,066 INFO    MainThread:1637445 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-15 16:51:28,066 INFO    MainThread:1637445 [wandb_init.py:init():765] run started, returning control to user process
2022-11-15 16:51:28,563 INFO    MainThread:1637445 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-15 16:51:28,563 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:52:09,552 INFO    MainThread:1637445 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-15 16:52:09,622 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 16:52:09,622 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 16:52:09,624 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 16:52:09,624 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 16:52:09,626 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 16:52:09,627 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 16:52:09,629 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165209-1r55bynf/logs/debug.log
2022-11-15 16:52:09,631 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165209-1r55bynf/logs/debug-internal.log
2022-11-15 16:52:09,632 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 16:52:09,632 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 16:52:09,632 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 2zw4zfjp
2022-11-15 16:52:09,637 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/2zw4zfjp
2022-11-15 16:52:09,637 INFO    MainThread:1637445 [jupyter.py:save_history():448] not saving jupyter history
2022-11-15 16:52:09,639 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:52:09,641 INFO    MainThread:1637445 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-11-15 16:52:09,641 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 16:52:09,643 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 16:52:09,645 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 16:52:14,153 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 16:52:14,154 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 16:52:14,162 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
